doc/NavigationService.md

```diff
@@ -1,6 +1,6 @@
 # Navigation Service
 ## Introduction
-The navigation ble service provides 4 characteristics to allow the the watch to display navigation instructions from a companion application.  The intended purpose is when performing some outdoor activities, for example running or cycling.
+The navigation ble service provides 4 characteristics to allow the the watch to display navigation instructions from a companion application. This service is intended to be used when performing some outdoor activities, for example running or cycling.
```

Comment:
There's another typo here "the the watch". Mind fixing that too?

Reply:
Thanks for catching that! Taking care of it.